docs/int/openapi.yaml

```diff
@@ -1106,7 +1296,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/JobCode'
-
+    MetaDataPatch:
+      type: array
+      items:
+        $ref: '#/components/schemas/JSONPatchRequestReplace'
+    JSONPatchRequestReplace:
```

Comment:
All request objects should be structured as JSON key-value pairs so it isn't necessary to include 'JSON' in the schema name. Should be 'MetadataUpdate' to match other component schemas and place this schema alphabetically in the components section, per current formatting.

- [ ] rename schema